kani-driver/src/args.rs

```diff
@@ -454,6 +478,14 @@ impl KaniArgs {
                 "The `--dry-run` option is obsolete. Use --verbose instead.",
             ));
         }
+        if let Some(out_dir) = &self.target_dir {
+            if out_dir.exists() && !out_dir.is_dir() {
+                return Err(Error::raw(
+                    ErrorKind::InvalidValue,
+                    "Invalid argument: `--target-dir` argument `{}` is not a directory",
```

Comment:
The argument (`out_dir`) is missing from the error message.

Reply:
And I'm missing the `format` call. That's why I need to add tests here.

Reply:
But I can't. :(

Reply:
Are you still planning on adding a `format` call?

Reply:
I can't believe I forgot the format. I must be going crazy. Sorry about that. Let me fix this

Reply:
Ah, I did add format somewhere else. I guess I just missed this one. Thanks for catching this.